src/LibraryManager.Vsix/Json/JsonHelpers.cs

```diff
@@ -65,7 +66,22 @@ internal static int FindInsertIndex(JSONParseItemList jsonParseItems, int rangeS
             return max + 1;
         }
 
-        public static bool TryGetInstallationState(JSONObject parent, out ILibraryInstallationState installationState, string defaultProvider = null)
+        internal static SortedNodeList<Node> GetChildren(Node node)
```

Comment:
Methods should avoid internal.  It should be public if intended to be called from outside of this class, or private if it is a local implementation detail.  The class is already marked internal.

Reply:
Updated this and other methods to be public